src/typescript/typescript.ts

```diff
@@ -430,9 +430,9 @@ export class TypeScriptProject extends NodeProject {
   private addJestNoCompile(jest: Jest) {
     this.addDevDeps("@types/jest", "ts-jest@^27"); // pinning for now because of an issue: https://github.com/projen/projen/issues/1813
 
-    jest.addTestMatch(`<rootDir>/${this.srcdir}/**/__tests__/**/*.ts?(x)`);
+    jest.addTestMatch(`**/${this.srcdir}/**/__tests__/**/*.ts?(x)`);
```

Comment:
```suggestion
    jest.addTestMatch(`${this.srcdir}/**/__tests__/**/*.ts?(x)`);
```

Is `**` needed as tests should only be inside `this.srcdir`